### ISSUE TITLE
There is no "pointer" cursor whe hovering over links

### ISSUE THREAD

**Issue by BAKFR** [jangernert/FeedReader#548]
When I move the cursor over a link, the default cursor icon (arrow) is used. 
I think the "pointer" icon to be used, as commonly used on every browser.

**Comment by BAKFR** [comment]
It doesn't seems to be a CSS problem.
I tried to override the `cursor` CSS property for link tags and I can see the result, but the `pointer` value display the default non-link cursor.

**Comment by jangernert** [comment]
Works for me. Both links as well as the custom css class:
```
.clickable-img-hover {
    outline: 2px solid black;
    cursor: pointer;
}
```

Do you by any chance use a custom cursor theme?

**Comment by BAKFR** [comment]
No, I use the default theme.
I've verified and changed it, and I've got some weird behavior.

When I change the theme, the pointer cursor became the pointer cursor of the default theme.
If I restart FeedReader, still with the custom theme, the displayed pointer cursor is the normal cursor of the custom theme. I tried a few theme. In some case, I've got the pointer cursor of the previously used theme.

I think it's the kind of bug hidden in the layers of gtk and/or webkit, hard to reproduce and yet more to find.
